src/SFA.DAS.ApprenticeCommitments/Data/Models/CourseDetails.cs

```diff
@@ -14,22 +14,23 @@ private CourseDetails()
 
         public CourseDetails(
             string name, int level, string? option,
-            DateTime plannedStartDate, DateTime plannedEndDate)
+            DateTime plannedStartDate, DateTime plannedEndDate,
+            int courseDuration)
         {
             Name = name;
             Level = level;
             Option = option;
             PlannedStartDate = plannedStartDate;
             PlannedEndDate = plannedEndDate;
+            CourseDuration = courseDuration;
         }
 
         public string Name { get; private set; } = null!;
         public int Level { get; private set; }
         public string? Option { get; private set; }
         public DateTime PlannedEndDate { get; private set; }
         public DateTime PlannedStartDate { get; private set; }
-
-        public int DurationInMonths => DifferenceInMonths(PlannedStartDate, PlannedEndDate);
+        public int CourseDuration { get; private set; }
```

Comment:
As a general guideline I prefer to either use a name that informs the unit of measurement (months in this case), or use a type that represents the unit (`TimeSpan` in this case).  Having a numeric field called simply "Duration", I don't think it's not immediately obvious that we are storing the number of months.  What do you think?